src/tags/object/Table.js

```diff
@@ -35,8 +35,16 @@ const Model = types
     get dataSource() {
       const { type } = parseTypeAndOption(self.valuetype);
 
+      const ordered = Object.keys(self._value).sort().reduce(
+        (obj, key) => {
+          obj[key] = self._value[key];
+          return obj;
+        },
+        {},
+      );
```

Comment:
Let's move this into the `if (type === 'json')` otherwise for array data it will cause errors.

Reply:
You're right Brandon, I already moved it into the if!